meerk40t/gui/propertypanels/textproperty.py

```diff
@@ -196,6 +203,16 @@ def pane_hide(self):
     def set_widgets(self, node):
         if node is not None:
             self.node = node
+        try:
+            if node.id is not None:
+                self.text_id.SetValue(str(node.id))
+        except AttributeError:
```

Comment:
Doesn't seem like it should need AttributeError catch here. If for some reason node.id, or node.label is missing it would be better to crash since we have bigger problems. We put those into the base node class. They should keep existing.